File: CPP/DPR.cpp
// Import Library
#include <iostream>
#include <string>

using namespace std;

class DPR{
    private:
        int id;
        string nama;
        string bidang;
        string partai;

    public:
        DPR(){
            this->id = 0;
            this->nama = "";
            this->bidang = "";
            this->partai = "";
        }    

        DPR(int id, string nama, string bidang, string partai){
            this->id = id;
            this->nama = nama;
            this->bidang = bidang;
            this->partai = partai;
        }

        int get_id(){
            return this->id;
        }

        void set_id(int id){
            this->id = id;
        }

        string get_nama(){
            return this->nama;
        }

        void set_nama(string nama){
            this->nama = nama;
        }

        string get_bidang(){
            return this->bidang;
        }

        void set_bidang(string bidang){
            this->bidang = bidang;
        }

        string get_partai(){
            return this->partai;
        }

        void set_partai(string partai){
            this->partai = partai;
        } 

        // void tambah(int i, int n){
        //     string nama;
        //     string bidang;       

        //     cin >> n;
        //     for(i = 0; i < n; i++){
        //         DPR temp;

        //         cin >> nama >> bidang;

        //         temp.set_nama(nama);
        //         temp.set_bidang(bidang);

        //         llist.push_back(temp);
        //     }
        // }



        // void eat(){
        //     cout << this->nama << " is eating!" << '\n';
        // }

        // void sleep(){
        //     cout << this->nama << " is sleeping!" << '\n';
        // }

        ~DPR(){

        }
};

File: PHP/DPR.php
<!-- Saya Rifa Sania NIM 2206697 mengerjakan Latihan Praktikum 1 dalam mata kuliah Struktur Data
untuk keberkahanNya maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin... -->

<?php

class DPR{
    private $id = 0;
    private $nama = '';
    private $bidang = '';
    private $partai = '';

    public function __construct($id = 0, $nama ='', $bidang = '', $partai = '') {
        $this->id = $id;
        $this->nama = $nama;
        $this->bidang  = $bidang;   
        $this->partai = $partai;    
    }

    public function setId($id){
        $this->id = $id;
    }

    public function getId(){
        return $this->id;
    }

    public function setNama($nama){
        $this->nama = $nama;
    }

    public function getNama(){
        return $this->nama;
    }

    public function setBidang($bidang){
        $this->bidang = $bidang;
    }

    public function getBidang(){
        return $this->bidang;
    }

    public function setPartai($partai){
        $this->partai = $partai;
    }

    public function getPartai(){
        return $this->partai;
    }

    // Fungsi untuk menambah data
    public function tambahData($id, $nama, $bidang, $partai) {
        global $data;
        $data[$id] = ['nama' => $nama, 'bidang' => $bidang, 'partai' => $partai];
    }

    // Fungsi untuk mengubah data
    public function ubahData($id, $nama, $bidang, $partai) {
        global $data;
        if (isset($data[$id])) {
            $data[$id] = ['nama' => $nama, 'bidang' => $bidang, 'partai' => $partai];
            echo "Data berhasil diubah.";
        } else {
            echo "ID tidak ditemukan.";
        }
    }

    // Fungsi untuk menghapus data
    public function hapusData($id) {
        global $data;
        if (isset($data[$id])) {
            unset($data[$id]);
            echo "Data berhasil dihapus.";
        } else {
            echo "ID tidak ditemukan.";
        }
    }

    public function tampilkanAnggotaDPR() {
        foreach ($this->$data as $anggota) {
            echo "ID: " . $anggota->getId() . ", Nama: " . $anggota->getNama() . ", Bidang: " . $anggota->getBidang() . ", Partai: " . $anggota->getPartai() . "<br>";
        }
    }
}    

?>